public/app/features/auth-config/fields.tsx

```diff
@@ -415,116 +498,158 @@ export function fieldMap(provider: string): Record<string, FieldData> {
+      label: t('auth-config.fields.allow-assign-grafana-admin-label', 'Allow assign Grafana admin'),
+      description: t(
+        'auth-config.fields.allow-assign-grafana-admin-description',
+        'If enabled, it will automatically sync the Grafana server administrator role.'
+      ),
       type: 'switch',
       hidden: !contextSrv.isGrafanaAdmin,
     },
     skipOrgRoleSync: {
-      label: 'Skip organization role sync',
-      description: 'Prevent synchronizing users’ organization roles from your IdP.',
+      label: t('auth-config.fields.skip-org-role-sync-label', 'Skip organization role sync'),
+      description: t(
+        'auth-config.fields.skip-org-role-sync-description',
+        "Prevent synchronizing users' organization roles from your IdP."
+      ),
       type: 'switch',
     },
     orgMapping: {
-      label: 'Organization mapping',
+      label: orgMappingLabel,
       description: orgMappingDescription(provider),
       type: 'select',
       hidden: !contextSrv.isGrafanaAdmin,
       multi: true,
       allowCustomValue: true,
       options: [],
-      placeholder: 'Enter mappings (my-team:1:Viewer...) and press Enter to add',
+      placeholder: t(
+        'auth-config.fields.organization-mapping-placeholder',
+        'Enter mappings (my-team:1:Viewer...) and press Enter to add'
+      ),
     },
     orgAttributePath: {
-      label: 'Organization attribute path',
-      description:
-        'JMESPath expression to use for organization lookup. If you configure "Organization mapping", you must also configure "Organization attribute path".',
+      label: orgAttributePathLabel,
+      description: t(
+        'auth-config.fields.organization-attribute-path-description',
+        'JMESPath expression to use for organization lookup. If you configure "{{ orgMappingLabel }}", you must also configure "{{ orgAttributePathLabel }}".',
+        { orgMappingLabel, orgAttributePathLabel }
+      ),
       type: 'text',
       hidden: !(['generic_oauth', 'okta'].includes(provider) && contextSrv.isGrafanaAdmin),
     },
     defineAllowedGroups: {
-      label: 'Define allowed groups',
+      label: t('auth-config.fields.define-allowed-groups-label', 'Define allowed groups'),
       type: 'switch',
     },
     defineAllowedTeamsIds: {
-      label: 'Define allowed teams ids',
+      label: t('auth-config.fields.define-allowed-teams-ids-label', 'Define allowed teams ids'),
       type: 'switch',
     },
     forceUseGraphApi: {
-      label: 'Force use Graph API',
-      description: "If enabled, Grafana will fetch the users' groups using the Microsoft Graph API.",
+      label: t('auth-config.fields.force-use-graph-api-label', 'Force use Graph API'),
+      description: t(
+        'auth-config.fields.force-use-graph-api-description',
+        "If enabled, Grafana will fetch the users' groups using the Microsoft Graph API."
```

Comment:
`Microsoft Graph API` shouldn't be translated as well. (I'm not sure if we need to create a separate label for it)

Reply:
I might leave this as-is in the string. Normally the translators know not to translate product names - it's just the very specific technical terminology that I wanted to make sure was sorted.